src/main/java/com/google/sps/servlets/EventServlet.java

```diff
@@ -95,4 +101,102 @@ private String getParameter(HttpServletRequest request, String name, String defa
     }
     return value;
   }
+
+  /** Format time to standard format. */
+  private String formatTime(String time) {
```

Comment:
Please consider use timestamp library like `java.time.*` instead of parsing string manually.

Reply:
Thank you, this was so useful! I formatted the time and date with SimpleDateFormat.